environments/staging/inventory.ini

```diff
@@ -125,12 +125,12 @@ redis2
 
 [elasticsearch:children]
 # Amazon EC2
-# old ES v2 hosts
-# es4
-# es5
-# ES v7 hosts
-es6
-es7
+# old ES v7.9.1 hosts
+es4
+es5
+# ES v7.6.2 hosts
+# es6
+# es7
```

Comment:
ES v7.9.1 is not the "old" version. Can you explain what's going on here?

Reply:
Will revert this, was planning to upgrade staging, but it's not necessary.